src/azure-firewall/azext_firewall/_params.py

```diff
@@ -113,6 +113,18 @@ def load_arguments(self, _):
         c.argument('public_ip_address', help='Name or ID of the public IP to use.', validator=get_public_ip_validator())
         c.argument('private_ip_address', deprecate_info=c.deprecate(expiration='2.3.0'), help='IP address used by the Firewall ILB as the next hop in User Defined Routes.')
 
+    with self.argument_context('network firewall ip-config') as c:
+        c.argument('management_item_name', options_list=['--m-name'], arg_group="Management Ip Config",
+                   help='Name of the management IP configuration.', is_preview=True)
+        c.argument('management_subnet', validator=get_management_subnet_validator(), help=argparse.SUPPRESS, is_preview=True)
+        c.argument('management_virtual_network_name', virtual_network_name_type, options_list=['--m-vnet-name'],
+                   arg_group="Management Ip Config",
+                   help='The virtual network (VNet) name for management ip configuation. '
+                        'It should contain one subnet called "AzureFirewallManagementSubnet".', is_preview=True)
+        c.argument('management_public_ip_address', help='Name or ID of the public IP to use for management ip configuation.',
+                   arg_group="Management Ip Config", options_list=['--m-public-ip-address'],
+                   validator=get_management_public_ip_validator(), is_preview=True)
```

Comment:
need `--m` prefix? Can we removed it? I don't know whether it will cause coflict?

Reply:
no. We already have --public-ip-address in ip-config create

Reply:
we have four almost same arguments. The only difference is the `m-` prefix.

Reply:
How about group them in a argument group **Management** to give user an instinct hint?

Reply:
Have done